modules/ducktests/tests/ignitetest/tests/sql_jdbc_test.py

```diff
@@ -0,0 +1,128 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This module contains SQL tests using the JDBC driver.
+"""
+from ducktape.mark.resource import cluster
+
+from ignitetest.services.ignite import IgniteService
+from ignitetest.services.utils.ignite_configuration import IgniteConfiguration
+from ignitetest.services.utils.sql_util import connection
+from ignitetest.utils import ignite_versions
+from ignitetest.utils.ignite_test import IgniteTest
+from ignitetest.utils.version import DEV_BRANCH, V_2_8_1, V_2_8_0, V_2_7_6, IgniteVersion
+
+
+# pylint: disable=W0223
+class SqlJdbcTest(IgniteTest):
+    """
+    SQL tests using the JDBC driver.
+    """
+    NUM_NODES = 3
+
+    @cluster(num_nodes=NUM_NODES)
+    # Note: DEV version does not support earlier versions before IGNITE-13414 resolution
+    @ignite_versions(
+        (str(DEV_BRANCH), str(DEV_BRANCH)),
+        (str(V_2_8_1), str(V_2_8_1)), (str(V_2_8_0), str(V_2_8_1)),
+        (str(V_2_8_1), str(V_2_8_0)), (str(V_2_8_0), str(V_2_8_0)),
+        (str(V_2_8_1), str(V_2_7_6)), (str(V_2_8_0), str(V_2_7_6)), (str(V_2_7_6), str(V_2_7_6))
+    )
+    def sql_test(self, ignite_version_1, ignite_version_2):
+        """
+        SQL test with different versions jdbc driver.
```

Comment:
What is a purpose of this test? If it checks that SQL works fine that why there are only CRUD operations, what about JOINs or complex queries. If it checks compatibility, is it required to run all that operations?

Reply:
Added alter tables and distributed join